packages/react-ui-core/package.json

```diff
@@ -31,6 +31,7 @@
     "babel-core": "6.26.0",
     "babel-eslint": "^8.2.2",
     "babel-jest": "^22.4.3",
+    "babel-plugin-dynamic-import-node": "^1.2.0",
```

Comment:
Is this actually working anywhere? I think we only removed an import.then, I'm not sure where it's happening